tests/monotouch-test/ImageIO/CGImageAnimationTest.cs

```diff
@@ -0,0 +1,157 @@
+﻿//
+// Unit tests for CGImageAnimation
+//
+// Authors:
+//	Whitney Schmidt <whschm@microsoft.com>
+//
+// Copyright 2020 Microsoft Corp. All rights reserved.
+//
+using System;
+using System.IO;
+#if XAMCORE_2_0
+using Foundation;
+using System.Threading.Tasks;
+#if MONOMAC
+using AppKit;
+#else
+using UIKit;
+#endif
+using CoreGraphics;
+using ImageIO;
+#else
+using MonoTouch.CoreGraphics;
+using MonoTouch.Foundation;
+using MonoTouch.ImageIO;
+using MonoTouch.UIKit;
+#endif
```

Comment:
are the MonoTouch namespaces needed? IIRC we no longer run tests using classic API

Reply:
removed MonoTouch namespaces! Lmk if any other namespaces here look like they are not needed, these using are copied from another ImageIO test.